[2025-04-06T18:15:00.266+0000] {processor.py:161} INFO - Started process (PID=149) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:15:00.267+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:15:00.267+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:15:00.267+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:15:00.382+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:15:00.380+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:15:00.382+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:15:00.391+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.127 seconds
[2025-04-06T18:15:30.552+0000] {processor.py:161} INFO - Started process (PID=151) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:15:30.554+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:15:30.556+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:15:30.556+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:15:30.760+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:15:30.759+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:15:30.760+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:15:30.771+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.230 seconds
[2025-04-06T18:16:01.070+0000] {processor.py:161} INFO - Started process (PID=153) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:16:01.071+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:16:01.072+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:16:01.072+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:16:01.196+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:16:01.195+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:16:01.196+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:16:01.206+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.141 seconds
[2025-04-06T18:16:31.560+0000] {processor.py:161} INFO - Started process (PID=155) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:16:31.562+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:16:31.562+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:16:31.562+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:16:31.740+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:16:31.739+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:16:31.740+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:16:31.750+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.201 seconds
[2025-04-06T18:17:04.420+0000] {processor.py:161} INFO - Started process (PID=149) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:17:04.420+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:17:04.421+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:17:04.421+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:17:04.515+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:17:04.514+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:17:04.515+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:17:04.523+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.105 seconds
[2025-04-06T18:17:34.684+0000] {processor.py:161} INFO - Started process (PID=151) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:17:34.686+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:17:34.688+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:17:34.687+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:17:34.826+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:17:34.825+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:17:34.826+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:17:34.836+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.160 seconds
[2025-04-06T18:18:05.199+0000] {processor.py:161} INFO - Started process (PID=154) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:18:05.201+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:18:05.204+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:18:05.204+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:18:05.381+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:18:05.380+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:18:05.381+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:18:05.393+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.206 seconds
[2025-04-06T18:18:29.303+0000] {processor.py:161} INFO - Started process (PID=148) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:18:29.303+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:18:29.304+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:18:29.304+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:18:29.404+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:18:29.403+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:18:29.404+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:18:29.413+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.113 seconds
[2025-04-06T18:18:44.305+0000] {processor.py:161} INFO - Started process (PID=151) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:18:44.306+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:18:44.307+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:18:44.307+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:18:44.435+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:18:44.434+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:18:44.435+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:18:44.444+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.144 seconds
[2025-04-06T18:19:14.594+0000] {processor.py:161} INFO - Started process (PID=153) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:19:14.595+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:19:14.597+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:19:14.596+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:19:14.718+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:19:14.717+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:19:14.718+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:19:14.728+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.142 seconds
[2025-04-06T18:20:05.122+0000] {processor.py:161} INFO - Started process (PID=148) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:20:05.123+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:20:05.124+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:20:05.123+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:20:05.223+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:20:05.223+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:20:05.224+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:20:05.233+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.114 seconds
[2025-04-06T18:20:35.407+0000] {processor.py:161} INFO - Started process (PID=152) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:20:35.409+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:20:35.411+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:20:35.411+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:20:35.583+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:20:35.582+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:20:35.583+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:20:35.596+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.200 seconds
[2025-04-06T18:21:06.053+0000] {processor.py:161} INFO - Started process (PID=154) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:21:06.055+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:21:06.056+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:21:06.056+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:21:06.257+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:21:06.254+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:21:06.257+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:21:06.272+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.232 seconds
[2025-04-06T18:22:12.576+0000] {processor.py:161} INFO - Started process (PID=149) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:22:12.576+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:22:12.577+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:22:12.577+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:22:12.697+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:22:12.696+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:22:12.697+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:22:12.706+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.133 seconds
[2025-04-06T18:22:43.849+0000] {processor.py:161} INFO - Started process (PID=152) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:22:43.850+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:22:43.852+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:22:43.852+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:22:43.997+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:22:43.996+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:22:43.997+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:22:44.008+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.167 seconds
[2025-04-06T18:23:14.167+0000] {processor.py:161} INFO - Started process (PID=154) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:23:14.169+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:23:14.172+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:23:14.171+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:23:14.393+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:23:14.392+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:23:14.393+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:23:14.406+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.260 seconds
[2025-04-06T18:39:14.564+0000] {processor.py:161} INFO - Started process (PID=150) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:39:14.565+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:39:14.566+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:39:14.566+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:39:14.692+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:39:14.691+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:39:14.692+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:39:14.702+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.141 seconds
[2025-04-06T18:39:44.806+0000] {processor.py:161} INFO - Started process (PID=152) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:39:44.807+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:39:44.810+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:39:44.809+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:39:45.008+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:39:45.007+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:39:45.009+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:39:45.025+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.225 seconds
[2025-04-06T18:40:15.438+0000] {processor.py:161} INFO - Started process (PID=154) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:40:15.439+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:40:15.440+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:40:15.439+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:40:15.579+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:40:15.577+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:40:15.580+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:40:15.587+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.158 seconds
[2025-04-06T18:40:45.929+0000] {processor.py:161} INFO - Started process (PID=156) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:40:45.930+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:40:45.931+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:40:45.931+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:40:46.068+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:40:46.066+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:40:46.068+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:40:46.078+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.158 seconds
[2025-04-06T18:41:16.481+0000] {processor.py:161} INFO - Started process (PID=158) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:41:16.483+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:41:16.485+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:41:16.484+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:41:16.628+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:41:16.626+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/pipeline.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/airflow/dags/pipeline.py", line 9, in <module>
    from airflow.providers.apache.hive.hooks.hive import HiveServer2Hook
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/hive/hooks/hive.py", line 38, in <module>
    from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
ModuleNotFoundError: No module named 'airflow.providers.common.compat'
[2025-04-06T18:41:16.628+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:41:16.638+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.168 seconds
[2025-04-06T18:53:19.623+0000] {processor.py:161} INFO - Started process (PID=148) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:53:19.624+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:53:19.625+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:53:19.625+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:53:19.732+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:53:19.735+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:53:19.735+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T18:53:19.740+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:53:19.740+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T18:53:19.754+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.133 seconds
[2025-04-06T18:53:50.913+0000] {processor.py:161} INFO - Started process (PID=152) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:53:50.915+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:53:50.919+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:53:50.918+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:53:51.173+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:53:51.236+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:53:51.236+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T18:53:51.241+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:53:51.241+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T18:53:51.254+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.357 seconds
[2025-04-06T18:54:21.889+0000] {processor.py:161} INFO - Started process (PID=196) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:54:21.891+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:54:21.893+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:54:21.892+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:54:22.037+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:54:22.080+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:54:22.080+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T18:54:22.084+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:54:22.084+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T18:54:22.095+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.215 seconds
[2025-04-06T18:54:53.110+0000] {processor.py:161} INFO - Started process (PID=198) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:54:53.113+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:54:53.116+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:54:53.115+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:54:53.318+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:54:53.395+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:54:53.395+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T18:54:53.402+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:54:53.402+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T18:54:53.416+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.315 seconds
[2025-04-06T18:55:23.897+0000] {processor.py:161} INFO - Started process (PID=200) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:55:23.898+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:55:23.900+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:55:23.900+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:55:24.044+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:55:24.087+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:55:24.087+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T18:55:24.091+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:55:24.091+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T18:55:24.102+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.211 seconds
[2025-04-06T18:55:54.376+0000] {processor.py:161} INFO - Started process (PID=202) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:55:54.378+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:55:54.380+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:55:54.379+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:55:54.508+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:55:54.549+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:55:54.549+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T18:55:54.554+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:55:54.554+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T18:55:54.565+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.200 seconds
[2025-04-06T18:56:24.761+0000] {processor.py:161} INFO - Started process (PID=204) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:56:24.762+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:56:24.763+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:56:24.762+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:56:24.899+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:56:24.943+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:56:24.942+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T18:56:24.947+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:56:24.947+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T18:56:24.957+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.201 seconds
[2025-04-06T18:56:55.277+0000] {processor.py:161} INFO - Started process (PID=206) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:56:55.279+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:56:55.284+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:56:55.283+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:56:55.578+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:56:55.633+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:56:55.633+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T18:56:55.638+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:56:55.638+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T18:56:55.652+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.402 seconds
[2025-04-06T18:57:25.853+0000] {processor.py:161} INFO - Started process (PID=208) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:57:25.855+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:57:25.857+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:57:25.856+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:57:26.030+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:57:26.083+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:57:26.083+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T18:57:26.089+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:57:26.088+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T18:57:26.100+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.252 seconds
[2025-04-06T18:57:56.334+0000] {processor.py:161} INFO - Started process (PID=210) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:57:56.336+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:57:56.344+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:57:56.338+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:57:56.527+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:57:56.586+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:57:56.586+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T18:57:56.591+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:57:56.591+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T18:57:56.602+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.279 seconds
[2025-04-06T18:59:12.374+0000] {processor.py:161} INFO - Started process (PID=149) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:59:12.375+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:59:12.376+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:59:12.376+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:59:12.588+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:59:12.592+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:59:12.592+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T18:59:12.599+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:59:12.599+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T18:59:12.621+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.252 seconds
[2025-04-06T18:59:43.563+0000] {processor.py:161} INFO - Started process (PID=164) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T18:59:43.564+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T18:59:43.566+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:59:43.565+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T18:59:43.734+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T18:59:43.825+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:59:43.825+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T18:59:43.830+0000] {logging_mixin.py:188} INFO - [2025-04-06T18:59:43.830+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T18:59:43.842+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.283 seconds
[2025-04-06T19:00:59.013+0000] {processor.py:161} INFO - Started process (PID=151) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T19:00:59.014+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T19:00:59.015+0000] {logging_mixin.py:188} INFO - [2025-04-06T19:00:59.015+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T19:00:59.198+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T19:00:59.202+0000] {logging_mixin.py:188} INFO - [2025-04-06T19:00:59.202+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T19:00:59.209+0000] {logging_mixin.py:188} INFO - [2025-04-06T19:00:59.208+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T19:00:59.222+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.213 seconds
[2025-04-06T19:01:29.864+0000] {processor.py:161} INFO - Started process (PID=195) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T19:01:29.866+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T19:01:29.869+0000] {logging_mixin.py:188} INFO - [2025-04-06T19:01:29.868+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T19:01:30.059+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T19:01:30.122+0000] {logging_mixin.py:188} INFO - [2025-04-06T19:01:30.121+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T19:01:30.128+0000] {logging_mixin.py:188} INFO - [2025-04-06T19:01:30.128+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T19:01:30.142+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.288 seconds
[2025-04-06T19:02:00.399+0000] {processor.py:161} INFO - Started process (PID=197) to work on /opt/airflow/dags/pipeline.py
[2025-04-06T19:02:00.401+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/pipeline.py for tasks to queue
[2025-04-06T19:02:00.405+0000] {logging_mixin.py:188} INFO - [2025-04-06T19:02:00.404+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/pipeline.py
[2025-04-06T19:02:00.670+0000] {processor.py:840} INFO - DAG(s) 'sensor_data_pipeline' retrieved from /opt/airflow/dags/pipeline.py
[2025-04-06T19:02:00.731+0000] {logging_mixin.py:188} INFO - [2025-04-06T19:02:00.731+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-04-06T19:02:00.736+0000] {logging_mixin.py:188} INFO - [2025-04-06T19:02:00.736+0000] {dag.py:3947} INFO - Setting next_dagrun for sensor_data_pipeline to None, run_after=None
[2025-04-06T19:02:00.749+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/pipeline.py took 0.363 seconds
